src/main/resources/templates/event-detail-view.html

```diff
@@ -153,8 +153,8 @@ <h1 class=" text-break display-1 fw-bolder text-white mb-2" th:text="${event.tit
                         <div sec:authorize="isAuthenticated()">
                             <div th:if="${event.startingDateTime.isAfter(#temporals.createNow())}">
                                 <!--Update event if Owner and If Future-->
-                                <div th:if="${#strings.equals(event.ownerNickname, loggedUser.username)}">
-                                    <a class="btn btn-success btn-lg px-4 me-sm-3"
+                                <div th:if="${#strings.equals(event.ownerNickname, loggedUser.username) || loggedUser.role.toString().equals('ROLE_ADMIN')}">
+                                    <a class=" btn btn-success btn-lg px-4 me-sm-3"
```

Comment:
Please, don't add unnecessary changes to our code. This space changes nothing so please remove it.

Reply:
this code is a need as it is responsible for displaying button the Update Event on the page...

Reply:
I'm referring to the space in line 157.